### PR TITLE
UserGuide for ActivityViewCommand and ActivityListCommand

### PR DESCRIPTION
**Pull request by lelerer**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -201,6 +203,63 @@ Example of usage:
...
+> <br> 
+Activities are a way for the user to keep track of the activities and their respective costs throughout a session.
+> <br>
+> Sessions are a way for the user to manage their gatherings that happen across the day.
+
+Format: `activity /view /sid [SESSION_ID] /aid [ACTIVITY_ID]`
+
+
+* `[SESSION_ID]` refers to the unique identifier of the session.
+    * The unique identifier for a session can be retrieved with `session /list` command.
```
Comment (froststein) [inline docs/UserGuide.md:218]: Perhaps add a hyperlink reference for the `session /list` command. Similar to what we did for the TOC.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -201,6 +203,63 @@ Example of usage:
...
+* `[ACTIVITY_ID]` refers to the unique identifier of the activity.
+    *  * The unique identifier for a session can be retrieved with `activity /list` command.
```
Comment (froststein) [inline docs/UserGuide.md:220]: Similarly you can add hyperlink reference as well for `activity /list`.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -201,6 +203,63 @@ Example of usage:
...
+
+> **💡 Note(s):**
+>- The session with a unique identifier of `[SESSION_ID]` and the activity with a unique identifier of `[ACTIVITY_ID]` have to exist before the activity can be viewed.
+
+
+Examples of usage:
+1. Views an activity with a unique identifier of 1 in a session with a session unique identifier of 1.
+    - `activity /view /sid 1 /aid 1` 
+<br>
+   ![Activity view command Screenshot](images/userguide/ActivityViewCommand.png)
```
Comment (froststein) [inline docs/UserGuide.md:230]: For the image link, please refer to other sections of the userguide to see how it is link.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -201,6 +203,63 @@ Example of usage:
...
+
+> **💡 Note(s):**
+>- The session with a unique identifier of `[SESSION_ID]` has to exist before the activities can be listed.
+
+
+Examples of usage:
+1. List the activities in a session with a unique identifier of 1.
+    - `activity /view /sid 1`
+      <br>
+      ![Activity list command Screenshot](images/userguide/ActivityListCommand.png)
```
Comment (froststein) [inline docs/UserGuide.md:257]: Same for here as well, for the image reference link.

File: docs/UserGuide.md (hunk 2 above)
Comment (warrencxw) [inline docs/UserGuide.md:220]: Consider removing the extra * for standardisation of indentation of the subpoint.

**Review by froststein — CHANGES_REQUESTED** [review]
Overall looks good, just need to add some link references and redo some existing references.

File: docs/UserGuide.md [hunk 5]
```diff
@@ -201,6 +203,63 @@ Example of usage:
...
+> **💡 Note(s):**
+>- The session with a unique identifier of `[SESSION_ID]` and the activity with a unique identifier of `[ACTIVITY_ID]` have to exist before the activity can be viewed.
+
+
+Examples of usage:
+1. Views an activity with a unique identifier of 1 in a session with a session unique identifier of 1.
+    - `activity /view /sid 1 /aid 1` 
+<br>
+   ![Activity view command Screenshot](images/userguide/ActivityViewCommand.png)
+      <br>
```
Comment (warrencxw) [inline docs/UserGuide.md:231]: Consider aligning this portion here as done in the section for activity list for easier readability.

File: docs/UserGuide.md [hunk 6]
```diff
@@ -201,6 +203,63 @@ Example of usage:
     ![Activity delete command Screenshot]()
 <br>
 <br>
+
+### Viewing an activity: `activity /view`
+> Allows user to view an existing activity in a particular session that is managed by SplitLah.
+> <br> 
+Activities are a way for the user to keep track of the activities and their respective costs throughout a session.
+> <br>
+> Sessions are a way for the user to manage their gatherings that happen across the day.
```
Comment (warrencxw) [inline docs/UserGuide.md:212]: Consider placing `<br>` behind the text and put text after the `>` symbol, as what others has done so to make it easier to read.

**Review by warrencxw — CHANGES_REQUESTED** [review]
Minor standardisation issues found, please rectify if possible.

File: docs/UserGuide.md [hunk 7]
```diff
@@ -201,6 +203,61 @@ Example of usage:
     ![Activity delete command Screenshot]()
 <br>
 <br>
+
+### Viewing an activity: `activity /view`
+> Allows user to view an existing activity in a particular session that is managed by SplitLah.
+> <br> 
+Activities are a way for the user to keep track of the activities and their respective costs throughout a session.
```
Comment (ivanaitzliddat) [inline docs/UserGuide.md:210]: Possible to add the > symbol in front of the sentence "Activities are a way ..."

**Review by ivanaitzliddat — CHANGES_REQUESTED** [review]
Looks good overall !! Added a comment to be rectified

File: docs/UserGuide.md [hunk 8]
```diff
@@ -215,36 +215,34 @@ Format: `activity /view /sid [SESSION_ID] /aid [ACTIVITY_ID]`
 
 
 * `[SESSION_ID]` refers to the unique identifier of the session.
-    * The unique identifier for a session can be retrieved with `session /list` command.
-* `[ACTIVITY_ID]` refers to the unique identifier of the activity.
-    *  * The unique identifier for a session can be retrieved with `activity /list` command.
+    * The unique identifier for a session can be retrieved with [`session /list`](#listing-all-sessions-session-list) command.
+* `[ACTIVITY_ID]` refers to the unique identifier of the activity. 
+    * The unique identifier for a session can be retrieved with [`activity /list`](#listing-all-activities-in-a-session-activity-list) command.
```
Comment (froststein) [inline docs/UserGuide.md:220]: The explanation for activity_id at the moment has a small mistake, it should be The unique identifier for a "session", it should not be session, it should be activity.

**Review by froststein — CHANGES_REQUESTED** [review]
Small errors on explanation for argument for ActivityViewCommand

**Review by froststein — APPROVED** [review]
Looks good to merge

**Review by warrencxw — APPROVED** [review]
Looks good to merge.

**Review by ivanaitzliddat — APPROVED** [review]
Looks good to merge

**Pull request merged**